### ISSUE TITLE
Add translation calls to ek-components

### ISSUE THREAD

**Issue by pwithnall** [endlessm/kolibri-explore-plugin#771]
Currently the ek-components package doesn’t fully support localisation/translation. It has a few translatable strings, but mostly hard-codes English strings in its code/templates.

**Comment by pwithnall** [comment]
One outstanding question is about the [privacy policy](https://github.com/endlessm/kolibri-explore-plugin/blob/master/packages/ek-components/src/components/EkPrivacyPolicyText.vue). This is a big bit of legal text, in English.

Do we want to translate it? From memory, for Endless OS, we had translated summaries of the policy, but the canonical legal text was always the English version. Do we want to do something similar for Endless Key?

**Comment by dsd** [comment]
Please just leave that as English-only for now

**Comment by pwithnall** [comment]
I have a [draft PR ready for review](https://github.com/endlessm/kolibri-explore-plugin/pull/777), but it’s blocked on some import issues which I can’t figure out.

**Comment by manuq** [comment]
Reviewed https://github.com/endlessm/kolibri-explore-plugin/pull/777#pullrequestreview-1613119748

> I have a [draft PR ready for review](https://github.com/endlessm/kolibri-explore-plugin/pull/777), but it’s blocked on some import issues which I can’t figure out.

Yeah, the first thing I have to say is.. sorry for the mess and unnecessary complexity of all this! We've been going back and forth with this project and this is the result.

I hope we can do this without adding kolibri as dependency. The ek-components are not intended to be used directly, but imported from the other components. The plugin should be the only one importing kolibri. The template-ui communicates with the plugin through [an API](https://github.com/endlessm/kolibri-explore-plugin/blob/master/kolibri_explore_plugin/assets/src/kolibriApi.js) and I was hoping to pass the translations through that API too in https://github.com/endlessm/kolibri-explore-plugin/issues/604. That leaves welcome-screen which is soon to be removed.

**Comment by manuq** [comment]
@pwithnall what about splitting the PR so we can merge the obvious translations like https://github.com/endlessm/kolibri-explore-plugin/pull/777/files#diff-0cf18e460b18be883d19e4be3bfeb50bc515f3e2c328f0e17c12bd7799fefed5L23-R108 now?

**Comment by pwithnall** [comment]
> @pwithnall what about splitting the PR so we can merge the obvious translations like https://github.com/endlessm/kolibri-explore-plugin/pull/777/files#diff-0cf18e460b18be883d19e4be3bfeb50bc515f3e2c328f0e17c12bd7799fefed5L23-R23 now?

I’ve split the first two commits out as #783.

**Comment by dbnicholson** [comment]
Finally landed this. I'd like to make sure we test the channel presentations to see if the ek-components strings that come through template-ui are rendered correctly.

**Comment by dbnicholson** [comment]
As discussed [here](https://github.com/endlessm/kolibri-explore-plugin/pull/840#discussion_r1333220955), the translations don't appear to be working. @pwithnall do you want to reopen this? It looks like you opened #846 to handle this issue.

**Comment by pwithnall** [comment]
#846 tracks the new issue. Reopening this would just lead to confusion over where to discuss the fix.

**Comment by vanessa-chang** [comment]
Verified Passed with flatpak v0.7 (Narwhal v.7.1.0)

It's not yet landed in the Windows version.
